### DISCUSSION TITLE
Weapon: Bimodal Weapon

### DISCUSSION THREAD

**Discussion by doubledh** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#435] [Ideas]
I want a weapon that is above-average and on use, swaps out to an alternate weapon (with distinct functionality and/or element) with the remaining durability.

Upgraded form is either prismatic (stronger and cycles through even more elements), or fused (no longer oscillates, and does both effects/elements)

**Comment by ntseng** [comment]
The math nerd in me wants a trimodal unupgraded weapon and then the three combinations of 3 pick 2 to be the upgrades, thus allowing the consistency to scale.

We might be able to get some UI complexity savings if the mode depended on public info such as weapon durability.
